towhee/hub/hub_tools.py

```diff
@@ -552,7 +581,8 @@ def generate_repo_yaml(author: str, repo: str) -> None:
         yaml.dump(data, outfile, default_flow_style=False, sort_keys=False)
 
 
-def main(argv):
+def main():
```

Comment:
The `main` function is so large,  we could use `argparse` lib to optimize it.

Reply:
working in progress #440 .